src/main/kotlin/me/owdding/lib/extensions/MapExtensions.kt

```diff
@@ -1,3 +1,8 @@
 package me.owdding.lib.extensions
 
 fun <K, V> MutableMap<K, V>.removeIf(predicate: (Map.Entry<K, V>) -> Boolean): MutableMap<K, V> = apply { entries.removeIf(predicate) }
+
+inline fun <Origin, Key, Value> Iterable<Origin>.associateNotNull(keySelector: (Origin) -> Key?, valueSelector: (Origin) -> Value?): Map<Key, Value> =
```

Comment:
we dont name any generics in here with full names, these should be single letters